nin/dasBoot/Node.js

```diff
@@ -0,0 +1,22 @@
+(function(NIN) {
+  'use strict';
+
+  class Node {
+    constructor(id, options) {
+      this.id = id;
+      this.inputs = options.inputs || {};
+      this.outputs = options.outputs || {};
+    }
+
+    resize() {
```

Comment:
One strategy is to throw in the base class. For non-vital methods, however, I'd rather they stay optional in case a node doesn't have anything to do in that method.